### DISCUSSION TITLE
Scenario transfer from RoadRunner to CARLA

### DISCUSSION THREAD

**Discussion by PhilippAuburger** [carla-simulator/carla#6856] [Q&A]
Hi all,

I have recently started to work with both RoadRunner and CARLA.
From RoadRunner it is possible to export maps, which can be imported in CARLA.

I would like to know if/how this can also be done for scenarios, i.e. including the driving behaviour of vehicles.

Thanks in advance.

**Comment by chris1202** [comment]
it can be done by exporting with openscenario format and importing it to carla with `make import ` command. there are good answers on the internet so search them for more info
